roles/lib_openshift/src/ansible/oc_secret.py

```diff
@@ -0,0 +1,37 @@
+# pylint: skip-file
+# flake8: noqa
+
+
+def main():
+    '''
+    ansible oc module for managing OpenShift Secrets
+    '''
+
+    module = AnsibleModule(
+        argument_spec=dict(
+            kubeconfig=dict(default='/etc/origin/master/admin.kubeconfig', type='str'),
+            state=dict(default='present', type='str',
+                       choices=['present', 'absent', 'list']),
+            debug=dict(default=False, type='bool'),
+            namespace=dict(default='default', type='str'),
```

Comment:
This `AnsibleModule` spec says the default for the `namespace` param is the **word** `default` and is of **TYPE** `string`, however the `DOCUMENTATION` says the default value for `namespace` is the **word** `str`:

```
  namespace:
    description:
    - The namespace where the object lives.
    required: false
    default: str
    aliases: []
```

I think the `DOCUMENTATION` just needs to be updated to say `default: default`.